src/main/java/cachiluis/model/Tutor.java

```diff
@@ -0,0 +1,4 @@
+package cachiluis.model;
+
+public class Tutor extends Person {
```

Comment:
An example will be that the Tutor(parent) has devices to which the notifications should be sent,